### PR TITLE
Pro 3096 pro 3114 tab configuration

### PR DESCRIPTION
**Pull request by RebeccaBaker**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-3096
https://tools.hmcts.net/jira/browse/PRO-3114

### Change description ###
Add new fields into request and response model as well as the transformer

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java [hunk 1]
```diff
@@ -145,6 +146,12 @@
 
     private final String solsFeeAccountNumber;
 
+    private final String casePrinted;
+
+    private final List<StopReasons> boCaseStopReasonList;
+
+    private final String boCaseStopReasons;
```
Comment (malinovsky) [inline src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java]: Do we need `boCaseStopReasons` field?

Comment (RebeccaBaker) [reply src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java]: Agreed, I have removed from the caseData.java and CallBackResponseTransformer.java

File: src/main/java/uk/gov/hmcts/probate/model/ccd/raw/response/ResponseCaseData.java [hunk 2]
```diff
@@ -65,4 +66,10 @@
     private final SolsAddress primaryApplicantAddress;
     private final List<AliasNames> solsDeceasedAliasNamesList;
     private final String ccdState;
+    private final String casePrinted;
+    private final String boEmailDocsReceivedNotification;
+    private final String boEmailGrantIssuedNotification;
+    private final String boCaseStopReasons;
```
Comment (malinovsky) [inline src/main/java/uk/gov/hmcts/probate/model/ccd/raw/response/ResponseCaseData.java]: Please remove unused `boCaseStopReasons` field as well.

File: src/main/java/uk/gov/hmcts/probate/model/ccd/raw/StopReason.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.probate.model.ccd.raw;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```
Comment (dhruvpatel20) [inline src/main/java/uk/gov/hmcts/probate/model/ccd/raw/StopReason.java]: You can remove @Data and @Builder from both classes.

Comment (dhruvpatel20) [reply src/main/java/uk/gov/hmcts/probate/model/ccd/raw/StopReason.java]: @AllArgsConstructor is enough.

Comment (RebeccaBaker) [reply src/main/java/uk/gov/hmcts/probate/model/ccd/raw/StopReason.java]: @Builder wasn't required, and changed @Data to just @Getter as we don't require the @Setter functionality

**Pull request merged**